tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/AntRunner.java

```diff
@@ -0,0 +1,179 @@
+package org.unicode.icu.tool.cldrtoicu.ant;
```

Comment:
The build failure is:
tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/ant/AntRunner.java
Above files did not contain the correct copyright notice. at tools/scripts/cpysearch/cpyscan.pl line 54.

Please prepend the following lines to this file:
```
// © 2020 and later: Unicode, Inc. and others.
// License & terms of use: http://www.unicode.org/copyright.html#License
```

Reply:
Ta. I was not necessarily expecting to submit this (more as a thing for Mark to copy into his testing utils). Happy to, but equally happy not to.